packages/server/server.ts

```diff
@@ -30,7 +30,11 @@ if (!PROD) {
 }
 
 uws
-  .App()
+  .SSLApp({
```

Comment:
interesting! did this let you do local debugging? 
(in production, our server sits behind nginx so SSL isn't needed at the app level)

Reply:
Yes, this was the way I could debug locally with https and successfully integrate with the callback url on AzDevops oAuth.

@jordanh , suggested for us to have this code based commited into the branch and deploy in a dev environment (so I presume behind nginx). @jordanh is also creating the app registration required for the azure Devops OAuth integration.

Then we would have a good dev environment to focus on the rest of the implementation.